kustomization.yaml

```diff
@@ -11,3 +11,19 @@ resources:
  - storage/overlays/gcp
 configurations:
  - configs/varreference.yaml
+configMapGenerator:
```

Comment:
@aleksandrmelnikov is the `configMapGenerator` supposed to be in root `kustomization.yaml` or is it better under `common/onepanel/base`? cc @Vafilor

Reply:
I think we placed it there so that `$(defaultNamespace)` could be used in the child kustomization.yaml files.